sdk/schemaregistry/azure-schemaregistry/swagger/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Schema Registry for Python
+
+> see https://aka.ms/autorest
+
+### Setup
+```ps
+cd C:\work
+git clone --recursive https://github.com/Azure/autorest.python.git
+cd autorest.python
+git checkout azure-core
+npm install
```

Comment:
Don't do any of those setup steps, it's deprecated

Reply:
Removed the setup section